src/test/java/com/marizueva/laboratory/hw3/fluentpageobject/pages/ServiceDifferentElementPage.java

```diff
@@ -0,0 +1,145 @@
+package com.marizueva.laboratory.hw3.fluentpageobject.pages;
+
+import org.openqa.selenium.NotFoundException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.testng.asserts.SoftAssert;
+
+import java.util.List;
+
+public class ServiceDifferentElementPage extends CommonPageElements {
+
+    @FindBy(css = "input[type='checkbox'")
+    List<WebElement> checkBoxes;
+
+    @FindBy(css = "input[type='radio'")
+    List<WebElement> radioButtons;
+
+    @FindBy(css = "div.colors")
+    List<WebElement> dropdownMenu;
+
+    @FindBy(css = ".uui-side-bar.right-fix-panel")
+    WebElement rightPanel;
+
+    //@FindBy(id = "mCSB_1_container")
+    //WebElement leftPanel;
+
+    @FindBy(xpath = "//label[@class='label-checkbox'][1]")
+    WebElement waterCheckbox;
+
+    @FindBy(xpath = "//label[@class='label-checkbox'][3]")
+    WebElement windCheckbox;
+
```

Comment:
Why do you use indexes in the xpath? This is not good practice because elements could be added before with the same class and your locator will be incorrect

Reply:
all xpath with indexes are eliminated